tests/unit/helpers/eventsRepetition.test.js

```diff
@@ -404,6 +437,32 @@ describe('createRepetitionsEveryWeekDay', () => {
       moment().range(moment('2019-01-11T09:00:00.000Z'), moment('2019-04-10T09:00:00.000Z')).by('days')
     );
 
+    dayDiff.returns(-3);
+
+    await EventsRepetitionHelper.createRepetitionsEveryWeekDay(event, sector);
+
+    sinon.assert.calledOnceWithExactly(
+      createRepeatedEvents,
+      event,
+      sinon.match(calledRange => JSON.stringify(calledRange) === JSON.stringify(range)),
+      sector,
+      true
+    );
+  });
+
+  it('should create repetition every week day from first event to current time plus 90 days', async () => {
```

Comment:
should create repetition every week day with first event in the past

Reply:
doen